cn.navclub.ldbfx.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "cn.navclub.ldbfx",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "ldbfx",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mariadb",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "ln -s /app/lib/mariadb/libmariadb.so /app/lib/libmariadb.so.3"
```

Comment:
This doesn't feel right, and is prone to break, given your app is dynamically linked to whatever SONAME version the connector provides. I would change it to just move files matching `libmariadb.so*` to /app/lib.